src/snapred/backend/dao/request/ReductionRequest.py

```diff
@@ -0,0 +1,16 @@
+from typing import List, Literal, Union
+
+from pydantic import BaseModel
+
+from snapred.backend.dao.state.FocusGroup import FocusGroup
+
+
+class ReductionRequest(BaseModel):
+    runNumber: str
+    useLitemode: bool
+    focusGroup: List[FocusGroup]
+    calibrantSamplePath: str
+
```

Comment:
We would probably also need a field to allow the user to provide a mask

Reply:
Would they provide it via a path?